### DISCUSSION TITLE
Relationships between Charts and Dashboards

### DISCUSSION THREAD

**Discussion by gauthk6** [lightdash/lightdash#6815] [Q&A]
Hi, is there any way to access the relationship between Charts and Dashboards using the Lightdash API? Like for instance say I have a dashboard called **dash**, which displays the following charts: **chart-1** and **chart-2**. Is there any way I can utilize the API such that it correctly returns me the information that **chart-1** and **chart-2** are being utilized by **dash**?

**Comment by magnew** [comment]
Hello @gauthk6! We do have this API, it's not yet part of the publicly publicly documented API, but I filed a ticket to clean annotate and document it: https://github.com/lightdash/lightdash/issues/6840

If you'd like to try it out before the documentation is ready, you are welcome to. The call pattern looks like 

`api/v1/dashboards/<dashboardUuId>`

And you will get a response that includes a `tiles` object which has all the charts. 

Does that seem like it will work for you?

**Comment by gauthk6** [comment]
That's perfect, thanks!
